src/services/services.ts

```diff
@@ -1643,7 +1643,7 @@ export function createLanguageService(
             getCancellationToken: () => cancellationToken,
             getCanonicalFileName,
             useCaseSensitiveFileNames: () => useCaseSensitiveFileNames,
-            getNewLine: () => getNewLineCharacter(newSettings, () => getNewLineOrDefaultFromHost(host)),
+            getNewLine: () => getNewLineCharacter(newSettings),

```

Comment:
Shouldnt this be `getNewLineOrDefaultFromHost` so it honors editor config ?

Reply:
I don't think so - the host here provides the setting through `host.getCompilationSettings()`

Reply:
Yeah, I think in this context, this is the compiler's new line setting, i.e. for emit, so this is correct.

Reply:
Oh, and this is just synchronnizeHostData, so there aren't any formatting settings here anyway, this is just plumbing host settings changes.